entries/c/carbonite.com.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Canva": {
```

Comment:
```suggestion
    "Carbonite": {
```
Thou shalt not copy-and-paste! :stuck_out_tongue_winking_eye:

Reply:
:)

Reply:
ups